ssc/cmod_windpower.cpp

```diff
@@ -285,6 +303,17 @@ void cm_windpower::exec()
 	}
 	wt.setPowerCurve(windSpeeds, powerOutput);
 
+    // get optional thrust curve for wind turbine
+    if (is_assigned("wind_turbine_ct_curve"))
+    {
+        size_t ctCurveLength = 0;
+        ssc_number_t* ctc = as_array("wind_turbine_ct_curve", &ctCurveLength);
+        std::vector<double> ct_curve(ctCurveLength);
+        for (size_t i = 0; i < ctCurveLength; i++)
+            ct_curve[i] = ctc[i];
+        wt.setCtCurve(ct_curve);
```

Comment:
Should there be an error or warning returned to the user if this function returns 0?